pkgs/development/libraries/relibc/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenvNoCC, buildPackages, fetchurl }:
+
+stdenvNoCC.mkDerivation {
+  name = "binary-relibc-latest";
+
+  # snapshot of https://static.redox-os.org/toolchain/x86_64-unknown-redox/relibc-install.tar.gz
+  src = fetchTarball {
```

Comment:
I think we have fetchipfs, re the eval failure

Reply:
As far as I know, fetchipfs requires a locally running ipfs daemon (not spawned by Nix), which I'd like to avoid

Reply:
Hmm, changing this to just `fetchurl` worked for me

Reply:
Hmm, changing this to just `fetchurl` works

Reply:
```suggestion
  src = fetchurl {
```

Reply:
noticed the hash was different also (when using fetchurl at least)
```
  wanted: sha256:0p1bxffbbl1bp8glwg2iqb18zjx678kyn91afjr5czlmcrrp3ybw
  got:    sha256:1hjdzrj67jdag3pm8h2dqh6xipbfxr6f4navdra6q1h83gl7jkd9

```

Reply:
Fixed. I also needed to add the following to avoid the error "unpacker produced multiple directories":
```
unpackPhase = "unpackFile $src";
```

Reply:
> As far as I know, fetchipfs requires a locally running ipfs daemon (not spawned by Nix), which I'd like to avoid

Maybe we should just fix that?

Reply:
> fetchipfs requires a locally running ipfs daemon

At least I'm fairly certain. I'll continue to try to figure out what's going on, then I'll open an issue.

In the meantime, though, I don't think fetchipfs issues should block this PR